# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/ilo/hotfix-cstdint.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ilo-r1.0.0/include/ilo/common_types.h.vanilla	2025-09-24 19:47:55.223127224 +0200
+++ ilo-r1.0.0/include/ilo/common_types.h	2025-09-24 19:48:04.834613076 +0200
@@ -95,6 +95,7 @@
 #include <memory>
 #include <string>
 #include <array>
+#include <cstdint>
 
 // Internal includes
 #include "ilo/version.h"
